Samples/2.0/ServiceFabric/Stateless/StatelessCalculatorService/StatelessCalculatorService.cs

```diff
@@ -37,21 +41,33 @@ protected override IEnumerable<ServiceInstanceListener> CreateServiceInstanceLis
             var listener = OrleansServiceListener.CreateStateless(
                 (serviceContext, builder) =>
                 {
+                    builder.Configure<ClusterOptions>(options =>
+                    {
+                        options.ServiceId = Guid.Empty;
+                        options.ClusterId = "dev";
+                    });
+
                     // Optional: use Service Fabric for cluster membership.
                     builder.UseServiceFabricClustering(serviceContext);
                     
                     // Optional: configure logging.
                     builder.ConfigureLogging(logging => logging.AddDebug());
 
-                    var config = new ClusterConfiguration();
-                    config.Globals.RegisterBootstrapProvider<BootstrapProvider>("poke_grains");
-                    config.Globals.ReminderServiceType = GlobalConfiguration.ReminderServiceProviderType.ReminderTableGrain;
+                    builder.AddStartupTask<StartupTask>();
+                    builder.UseInMemoryReminderService();
 
                     // Service Fabric manages port allocations, so update the configuration using those ports.
-                    config.Defaults.ConfigureServiceFabricSiloEndpoints(serviceContext);
+                    builder.Configure<EndpointOptions>(options =>
```

Comment:
Considering `builder.ConfigureEndpoints` is used, using `builder.Configure<EndpointOptions>` might be unnecessary?

Reply:
Yes, you're right, thank you